### PR TITLE
chore: add metrics spec

### PR DESCRIPTION
**Pull request by katcipis**
Specify metrics we want to export on the event library. PR with code will follow.

### CODE REVIEW

File: event/README.md [hunk 1]
```diff
@@ -3,3 +3,52 @@
...
+Total of published messages.
+
+Labels:
+
+* status : "ack" or "nack".
+* name : name of the event.
+
+### Subscription
+
+#### event_process_duration_seconds : histogram
```
Comment (katcipis) [inline event/README.md:38]: Not a huge fan of this metric name x_x.

**Review by guilherme-santos — DISMISSED** [review]
🚀

File: event/README.md (hunk 1 above)
Comment (guilherme-santos) [reply event/README.md:38]: What bothers you? we could use publish/consume WDYT?

**Review by matmarzolagomes — APPROVED** [review]

File: event/README.md (hunk 1 above)
Comment (katcipis) [reply event/README.md:38]: Not sure what was bothering me..but was feeling off with the name :laughing: . I thought about consume but it also didn't made me happy...maybe nothing will made me happy here :laughing: 

WDYT @lfritz ?

Comment (lfritz) [reply event/README.md:38]: This is basically the time it takes until we can ack or nack the message, right? Consume and process both seem fine for that case, in my mind :smile:

File: event/README.md [hunk 2]
```diff
@@ -3,3 +3,52 @@
...
+### Publisher
+
+#### event_publish_duration_seconds : histogram
+
+Measure publish duration time for each event.
+Useful when debugging connectivy issues between a service and the message broker.
+
+Labels:
+
+* status : "ack" or "nack".
```
[the remark this answers is not in the document]
Comment (katcipis) [reply event/README.md:24]: yeah I tried to be more event/pubsub specific but it doesn't fit well with status (unless we rename the label).

File: event/README.md (hunk 1 above)
Comment (katcipis) [reply event/README.md:38]: will leave as process for now

**Review by guilherme-santos — APPROVED** [review]
👍🏻

**Pull request merged**